auth_supplier/__manifest__.py

```diff
@@ -7,7 +7,7 @@
 {
     'name': "Auth Supplier",
     'category': 'Portal',
-    'version': '9.0.2.0.0',
+    'version': '10.0.1.0.0',
```

Comment:
Oh, missed that one, sorry. I would like to read the discussion on how this decision came up if you have a link.